db/model/Gdoc/extractGdocComponentInfo.ts

```diff
@@ -68,11 +68,15 @@ function iterateArrayProp<T extends OwidEnrichedGdocBlock>(
     parentPath: string,
     prop: keyof T
 ): ChildIterationInfo[] {
-    return (parent[prop] as OwidEnrichedGdocBlock[]).map((child, index) => ({
-        child: child,
-        parentPath: `${parentPath}`,
-        path: `${parentPath}.${String(prop)}[${index}]`,
-    }))
+    if (prop in parent)
```

Comment:
This will be true, if the property is present, but has `undefined` as the value. We probably want:

```suggestion
    if (parent[prop])
```

Reply:
Ah right, thanks for catching that!